app/assets/stylesheets/content/_in_place_editing.sass

```diff
@@ -47,14 +47,21 @@ $inplace-edit--color--very-dark: #cacaca
       cursor: wait!important
       inplace-edit--write
   .inplace-edit--write-value
-    .date-range-picker
+    .inplace-edit--date-range
```

Comment:
@hirtie-maxim we generally avoid nesting selectors (unless you actually want the behaviour to differ in particular contexts)